mimic/rest/loadbalancer_api.py

```diff
@@ -54,10 +57,104 @@ def resource_for_region(self, region, uri_prefix, session_store):
 
+    def _get_session(self, session_store, tenant_id):
+        """
+        Retrieve or create a new LoadBalancer session from a given tenant identifier
+        and :obj:`SessionStore`.
+
+        For use with ``data_for_api``.
+
+        Temporary hack; see this issue
+        https://github.com/rackerlabs/mimic/issues/158
+        """
+        return (
+            session_store.session_for_tenant_id(tenant_id)
+            .data_for_api(self, lambda: GlobalServerCollections(
+                tenant_id=tenant_id,
+                clock=session_store.clock
+            ))
+        )
+
+
+@implementer(IAPIMock, IPlugin)
+@attributes(["lb_api"])
+class LoadBalancerControlApi(object):
+    """
+    This class registers the load balancer controller API in the service
+    catalog.  It does NOT implement individual endpoints.
+    """
+    def catalog_entries(self, tenant_id):
+        """
+        Cloud load balancer controller endpoints.
+        """
+        return [
+            Entry(
+                tenant_id, "rax:load-balancer", "cloudLoadBalancerControl",
+                [
+                    Endpoint(tenant_id, region, text_type(uuid4()), prefix="v2")
+                    for region in self.lb_api._regions
+                ]
+            )
+        ]
+
+    def resource_for_region(self, region, uri_prefix, session_store):
+        """
+        Get an :obj:`twisted.web.iweb.IResource` for the given URI prefix;
+        implement :obj:`IAPIMock`.
+        """
+        lbc_region = LoadBalancerControlRegion(api_mock=self, uri_prefix=uri_prefix,
+                                               session_store=self.lb_api.session_store, region=region)
+        return lbc_region.app.resource()
+
+
+@attributes(["api_mock", "uri_prefix", "session_store", "region"])
+class LoadBalancerControlRegion(object):
+    """
+    Klein routes for load balancer's control API within a particular region.
+    """
+
+    app = MimicApp()
+
+    def session(self, tenant_id):
+        """
+        Gets a session for a particular tenant, creating one if there isn't
+        one.
+        """
+        tenant_session = self._session_store.session_for_tenant_id(tenant_id)
+        clb_global_collection = tenant_session.data_for_api(
+            self._api_mock,
+            lambda: GlobalCLBCollections(
+                tenant_id=tenant_id,
+                clock=self._session_store.clock))
+        clb_region_collection = clb_global_collection.collection_for_region(
+            self.region_name)
+        return clb_region_collection
+
+    def _collection_from_tenant(self, tenant_id):
+        """
+        Retrieve the server collection for this region for the given tenant.
+        """
+        return (self.api_mock.lb_api._get_session(self.session_store, tenant_id)
+                .collection_for_region(self.region))
+
+    @app.route('/v2/<string:tenant_id>/loadbalancer/<string:clb_id>/returnOverride/<int:statusCode>', methods=['POST'])
```

Comment:
The load balancer API specifies `lb_id` as an int, rather than a string, so the storage dictionary is keyed by ints, rather than strings.  In order to make that match, maybe this should be changed to `<int:clb_id>`?